check_stability.py

```diff
@@ -353,6 +353,53 @@ def get_files_changed():
             for item in files[:-1].split("\0")]
 
 
+def get_affected_testfiles(files_changed):
+    affected_testfiles = []
+    ignore_dirs = ["conformance-checkers", "docs"]
+    manifest_file = os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests", "MANIFEST.json")
+    with open(manifest_file, "r") as fh:
+        manifest_contents = fh.read()
+    for changedfile_pathname in files_changed:
+        repo_root = os.path.abspath(os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests"))
+        changed_file_repo_path = changedfile_pathname[len(repo_root):]
+        if changed_file_repo_path in manifest_contents:
```

Comment:
(or, reading some later code, just do

    manifest_files = {os.path.join(repo_root, item.path) for item in manifest.itertypes("reftest", "testharness", [etc.])}

where the point of enumerating the test types is that the new manifest will also include support files. Of course once that lands we can just check the file is in the list of support files and so things will get simpler.)